### PR TITLE
Add live-2 cluster to the production standard clusters list

### PR DESCRIPTION
**Pull request by poornima-krishnasamy**
-  add `live-2` to the list of production clusters listed in this repo. This PR include live-2 as production for monitoring and opa modules which differ from test clusters.
- does-not include `live-2` to production list for ingress-controllers, cert-manager, and external-dns modules

Related to: https://github.com/ministryofjustice/cloud-platform/issues/4101

### CODE REVIEW

File: terraform/aws-accounts/cloud-platform-aws/vpc/eks/components/components.tf [hunk 1]
```diff
@@ -105,7 +106,7 @@ module "modsec_ingress_controllers_v1" {
   replica_count       = "6"
   controller_name     = "modsec"
   cluster_domain_name = data.terraform_remote_state.cluster.outputs.cluster_domain_name
-  is_live_cluster     = lookup(local.prod_workspace, terraform.workspace, false)
+  is_live_cluster     = terraform.workspace == "live" ? true : false
```
Comment (sablumiah) [inline terraform/aws-accounts/cloud-platform-aws/vpc/eks/components/components.tf:109]: Should `live-2` also be true for ``is_live_cluster?`

File: terraform/aws-accounts/cloud-platform-aws/vpc/eks/components/main.tf [hunk 2]
```diff
@@ -80,22 +80,23 @@ data "aws_route53_zone" "cloud_platform" {
...
   }
 
-  cloudwatch_workspace = {
-    manager = false
+  # live_workspace refer to all production workspaces which have users workload in it
+  live_workspace = {
     live    = true
     live-1  = true
     default = false
   }
```
Comment (sablumiah) [inline terraform/aws-accounts/cloud-platform-aws/vpc/eks/components/main.tf:96]: So we need to add live-2 here in live_workspace?

File: terraform/aws-accounts/cloud-platform-aws/vpc/eks/components/components.tf (hunk 1 above)
Comment (poornima-krishnasamy) [reply terraform/aws-accounts/cloud-platform-aws/vpc/eks/components/components.tf:109]: This flag is used to determine external-dns-annotation to serve hosts with `*.cloud-platform.service.justice.gov.uk`. I dont think we share the same hosts under *.cloud-platform. between 2 clusters. So I have left the live-2 here.

File: terraform/aws-accounts/cloud-platform-aws/vpc/eks/components/main.tf (hunk 2 above)
Comment (poornima-krishnasamy) [reply terraform/aws-accounts/cloud-platform-aws/vpc/eks/components/main.tf:96]: Thanks Sablu. Thats correct. Have added.

**Review by sablumiah — APPROVED** [review]

**Review by vijay-veeranki — APPROVED** [review]

**Pull request merged**